AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -58,6 +20,10 @@ typedef struct {
   ASStackLayoutJustifyContent justifyContent;
   /** Orientation of children along cross axis */
   ASStackLayoutAlignItems alignItems;
+  /** 
+   If YES the vertical spacing between two views is measured from the last baseline of the
+   top view to the top of the bottom view*/
+  BOOL baselineRelativeArrangement;
```

Comment:
Do we still need this? I don't think it's used anywhere. Even if it is needed, I think it belongs to ASBaselineStackLayoutSpec.